drivers/gpio/gpio_pca953x.c

```diff
@@ -467,7 +479,9 @@ static const struct gpio_driver_api api_table = {
 			.port_pin_mask = GPIO_PORT_PIN_MASK_FROM_DT_INST(n),	\
 		},								\
 		.interrupt_enabled = DT_INST_NODE_HAS_PROP(n, nint_gpios),	\
-		.gpio_int = GPIO_DT_SPEC_INST_GET_OR(n, nint_gpios, {0}),	\
+		.gpio_int = GPIO_DT_SPEC_INST_GET_OR(n, nint_gpios, {0}),  \
+		.interrupt_mask = DT_INST_PROP_OR(n, interrupt_mask, -1),	\
+		.input_latch = DT_INST_PROP_OR(n, input_latch, -1)		\
```

Comment:
clang-format will work properly for these if you add a comma at the end, i.e.

```c
.input_latch = DT_INST_PROP_OR(n, input_latch, -1),
```

Reply:
Thank you for your comment, has been updated!